synfig-studio/src/gui/widgets/widget_link.h

```diff
@@ -40,18 +42,48 @@ namespace studio {
 
 class Widget_Link: public Gtk::ToggleButton
 {
-	synfig::String tooltip_inactive_;
-	synfig::String tooltip_active_;
+	std::string tooltip_inactive_;
+	std::string tooltip_active_;
 
 	Gtk::Image *icon_off_;
 	Gtk::Image *icon_on_;
 
+	void init();
+	void init(const std::string &tlt_inactive, const std::string &tlt_active);
+	void set_up_icons();
+
 protected:
 	void on_toggled();
 
 public:
+	Widget_Link();
 	Widget_Link(const std::string &tlt_inactive, const std::string &tlt_active);
 	~Widget_Link();
+
+// Glade & GtkBuilder related
+private:
+	Glib::RefPtr<Gtk::Builder> builder;
+
+	const bool IS_WIDGET_FROM_BUILDER;
+
+	static GType gtype;
+
+	void refresh_tooltip_text();
+
+public:
+	Widget_Link(BaseObjectType *cobject);
+	Widget_Link(
+			BaseObjectType *cobject, const Glib::RefPtr<Gtk::Builder> &builder,
+			const std::string tlt_inactive, const std::string tlt_active);
+
+	static Glib::ObjectBase *wrap_new(GObject *o);
+	static void register_type();
+
+	Glib::Property<std::string> value_active;
+	Glib::Property<std::string> value_inactive;
+
+	Glib::PropertyProxy<std::string> property_tooltip_active()   { return value_active.get_proxy(); }
```

Comment:
If you use proxy, make the previous lines private :)

Reply:
Fixing it...

Reply:
Done